crawler.py

```diff
@@ -353,15 +367,29 @@ def crawl(self):
         a virtual browser with Privacy Badger installed. Afterwards, save the
         action_map and snitch_map that the Badger learned.
         """
-        domains = get_domain_list(self.n_sites)
-        self.logger.info((
-            "starting new crawl:\n"
-            "\ttimeout: %ss\n"
-            "\twait time: %ss\n"
-            "\tbrowser: %s\n"
-            "\tsurvey mode: False\n"
-            "\tdomains to crawl: %d"
-        ), self.timeout, self.wait_time, self.browser, self.n_sites)
+        
+        # generate domain list based on n_sites and excluded TLDs
+        domains = get_domain_list(self.n_sites, self.exclude)
+        # only print statement about excluded TLDs if that option is passed in
+        if self.exclude:
```

Comment:
This if/else is too much duplication. What if we had another optional setting that we wanted to log here? Let's simplify ... How about always printing out `"\tTLDs to exclude: %s"`?

Reply:
Good call